app/views/navbar.scala.html

```diff
@@ -29,10 +29,26 @@
                     <li>
                         <a href="#" id="toolbar-onboarding-link">Retake tutorial</a>
                     </li>
-                }
-
-
 
+                }
+                @if(url.get == "/audit"){
+                    <script type="text/javascript">
```

Comment:
I can put the css elements in a class but I need to leave the @if and script statement there to only use the custom css on the audit interface.